src/components/calcite-input/calcite-input.tsx

```diff
@@ -512,7 +512,6 @@ export class CalciteInput implements LabelableComponent {
     if (direction === "up" && ((!inputMax && inputMax !== 0) || inputVal < inputMax)) {
       newValue = (inputVal + inputStep).toString();
     }
-
```

Comment:
Please revert this as it is not related to the fix. 

Sidebar: IMO, newlines between if statements help with readability (easier to scan vertically).